src/main/kotlin/uk/gov/gdx/datashare/controllers/EventsController.kt

```diff
@@ -156,7 +156,6 @@ class EventsController(
     @Schema(
       description = "Event Types, if none supplied it will be the allowed types for this client",
       required = false,
-      allowableValues = ["DEATH_NOTIFICATION", "LIFE_EVENT"],
```

Comment:
Why remove this all over?

Reply:
Because its now an enum and you get it for free!

Reply:
Huh I didn't see it in swagger for free...